examples/documentation_examples/examples.go

```diff
@@ -2097,3 +2097,200 @@ func ChangeStreamExamples(t *testing.T, db *mongo.Database) {
 
 	atomic.StoreInt32(&stop, 1)
 }
+
+// AggregationExamples contains examples of aggregation operations.
+func AggregationExamples(t *testing.T, db *mongo.Database) {
+	ctx := context.Background()
+
+	coll := db.Collection("sales")
+	coll1 := db.Collection("air_alliances")
+
+	err := coll.Drop(context.Background())
+	require.NoError(t, err)
+	err1 := coll1.Drop(context.Background())
+	require.NoError(t, err1)
+
+	{
+		// Start Aggregation Example 1
+		pipeline := bson.A{
```

Comment:
It's more verbose, but it might be more readable if there were more newlines: https://play.golang.org/p/hmA8o6B4mV6. I'll only put this comment here, but it applies to each pipeline declaration in the examples.

Reply:
Sounds good, thanks for the example! I updated all the pipelines to be more readable; let me know if I can improve the readability in any other way.